api/apps/geoprocessing/src/marxan-sandboxed-runner/marxan-run.ts

```diff
@@ -37,20 +37,36 @@ export class MarxanRun extends MarxanRunEmitter implements Cancellable {
     this.#process = spawn(workspace.marxanBinaryPath, {
       cwd: workspace.workingDirectory,
     });
-
     assertDefined(this.#process);
-
     this.#process.stderr.on('data', (chunk) => {
       this.#stdError.push(chunk.toString());
     });
-
     const progressWatcher = new Progress();
     this.#process.stdout.on('data', (chunk) => {
       const currentProgress = progressWatcher.read(chunk);
       this.emit(`progress`, currentProgress);
       this.#stdOut.push(chunk.toString());
     });
 
+    this.#process.on(
+      `error`,
+      (
+        error: Error & {
+          errno: number;
+          code: number;
+          path: string;
+          syscall: string;
+          spawnargs: string[];
+        },
```

Comment:
```suggestion
        error: Error | {
          errno: number;
          code: number;
          path: string;
          syscall: string;
          spawnargs: string[];
        },
```
otherwise the `code` in error doesn't make sense